packages/x-components/src/x-modules/facets/components/filters/simple-filter.vue

```diff
@@ -130,6 +148,16 @@ property as text.
 </script>
 ```
 
+### Playing with props
+
+Configuring the events to emit when the filter is clicked.
+
+```vue
+<template>
+  <SimpleFilter :clickEvents="{ UserClickedASimpleFilter: filter }" :filter="filter" />
+</template>
```

Comment:
Complete this example with the <script> section, we want to have examples that you can copy and paste. :kekw: